frameworks/non-keyed/sifrr/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "js-framework-benchmark-stage0",
```

Comment:
Man, I'm really glad to be your inspiration but... come on.

Reply:
Sorry 🙇